spatialprofilingtoolbox/graphs/scripts/generate_graphs.py

```diff
@@ -0,0 +1,53 @@
+"""Generate graphs from SPT data extracts."""
```

Comment:
Here and elsewhere you refer to "SPT data extracts" and similar. What does this mean? Isn't it really specifications for graph generation? It's not "from" SPT in any sense, and it's also not really data, more like function arguments. Is this understanding correct?

Reply:
This doctoring is describing what the CLI does, but I take it you want it to describe what's in the Python file. This can be fixed.

Reply:
I see, my understanding was not correct. I didn't realize that the graph creation from database extracts was separate from generating those extracts from the db.